### ISSUE TITLE
Error messages for zip code and email address differs between org model and non-org model instances

### ISSUE THREAD

**Issue by lizpearl** [cisagov/manage.get.gov#3087]
### Issue description

Recently error messages were changed for zip codes and email address for org-model.  We need to make sure the changes are also made to non org model instances.

If it makes sense, also look into centralizing the error messages.

### Acceptance criteria
These should already be done for org model and needs to be done for non-org model cases
- [x] On the "Organization" page:
  - [x] When zip code is required and left blank, update error message to read: 
    - Enter a 5-digit or 9-digit zip code, like 12345 or 12345-6789.
  - [x] When zip code uses wrong format, update the error message to match the message listed above. (we want them to be consistent)
- [x] On the "add a domain manager" page:
  - [x] When email address is required and left blank, update error message to read: 
    - Enter an email address in the required format, like name@example.com.
  - [x] The error message should appear at the top of the page and at the field level. Currently, it's only displayed at the field level.
    - This should be updated for org model AND non-org model
- [x] Review registrar and verify other instances where these fields appear show this updated messaging

### Additional context
See parent ticket, especially [comment from the parent ticket](https://github.com/cisagov/manage.get.gov/issues/2777#issuecomment-2471296151)
